### ISSUE TITLE
Release v0.57.2

### ISSUE THREAD

**Issue by jpkrohling** [open-telemetry/opentelemetry-collector#5769]
Like open-telemetry/opentelemetry-collector#4522, but for ~~v0.57.0~~ v0.57.2

Versions v0.57.0 and v0.57.1 had problems during the release: first, the tag didn't seem to reflect the actual "chore" commit, and then I just retagged 0.57.1, which wasn't the right thing to do.

**Performed by collector release manager**

- [x] open-telemetry/opentelemetry-collector#5771
- [x] open-telemetry/opentelemetry-collector#5772
- [x] open-telemetry/opentelemetry-collector#5773
- [x] open-telemetry/opentelemetry-collector#5774
- [x] open-telemetry/opentelemetry-collector#5775
- [x] open-telemetry/opentelemetry-collector#5776

**Performed by operator maintainers**

- [ ] open-telemetry/opentelemetry-operator#1016

**Performed by helm chart maintainers**

- [x] open-telemetry/opentelemetry-helm-charts#300

**Comment by jpkrohling** [comment]
The release is currently blocked on:

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12661
https://github.com/open-telemetry/opentelemetry-collector/issues/5770

**Comment by amoscatelli** [comment]
@jpkrohling 
I think both issues are merged now

**Comment by jpkrohling** [comment]
One more blocker: https://github.com/open-telemetry/opentelemetry-collector/pull/5797

**Comment by bogdandrutu** [comment]
Last item has a separate issue, ok to close this.
